amti/clis/associate.py

```diff
@@ -72,8 +72,8 @@ def associate_qual(file, ids, qual, name, value, notify, live):
         "QualificationTypeId": qual_id,
         "SendNotification": notify
     }
-    if value is not None:
-        args['IntegerValue'] = value
+    if integer_value is not None:
+        args['IntegerValue'] = int(integer_value)
```

Comment:
With the change suggested above (using `type=int` in the click option) it will no longer be necessary to coerce to an int here.